sections/header.liquid

```diff
@@ -728,8 +745,8 @@
     "@context": "http://schema.org",
     "@type": "Organization",
     "name": {{ shop.name | json }},
-    {% if section.settings.logo %}
-      "logo": {{ section.settings.logo | image_url: width: 500 | prepend: "https:" | json }},
+    {% if settings.logo %}
+      "logo": {{ settings.logo | image_url: width: 500 | prepend: "https:" | json }},
```

Comment:
Yeah. It seems it really had been forgotten there just because this chunk of code is responsible for sharing some data to Schema.org to improve SEO experience and it doesn't affect a functionality of the page.